politeiawww/api/cms/v1/v1.go

```diff
@@ -992,3 +994,27 @@ type CastVoteReply struct {
 	Error           string                 `json:"error"`                 // Error status message
 	ErrorStatus     cmsplugin.ErrorStatusT `json:"errorstatus,omitempty"` // Error status code
 }
+
+type ProposalBillingSummary struct {
```

Comment:
Now I noticed that there's this `struct` defined but it doesn't seem like I can send `start` and `end` params (at least it's not documented).

Reply:
Whoops sorry that was an early addition and forgot to remove.  Thanks for pointing out.  Removed.